daemon/algod/api/server/v2/utils.go

```diff
@@ -206,16 +205,16 @@ func computeAppIndexFromTxn(tx node.TxnWithStatus, l LedgerForAPI) *uint64 {
 }
 
 // getCodecHandle converts a format string into the encoder + content type
-func getCodecHandle(formatPtr *string) (codec.Handle, string, error) {
-	format := "json"
+func getCodecHandle(formatPtr *model.Format) (codec.Handle, string, error) {
+	format := model.Json
 	if formatPtr != nil {
-		format = strings.ToLower(*formatPtr)
+		format = model.PendingTransactionInformationParamsFormat(*formatPtr)
```

Comment:
*formatPtr doesn't need to be lowered cased anymore?

Reply:
I added it back in the most recent commits. It looks like it got removed when the method was changed previously, but you're right we still want to be case insensitive here.